### DISCUSSION TITLE
how to deal with “clust01（tcl）*ERROR* SIGABRT” problem

### DISCUSSION THREAD

**Discussion by yMichaelqh** [The-OpenROAD-Project/OpenROAD-flow-scripts#1633] [Q&A]
when i execute Regression Tests，there were 2errors，if there are solutions to solve this problem，thank you
![Uploading 捕获.PNG…]()

**Comment by maliberty** [comment]
Please open this as an issue with all the details.  There isn't enough info here to do anything.  Also the png link is broken.
